fdbserver/masterserver.actor.cpp

```diff
@@ -1242,6 +1242,13 @@ ACTOR Future<Void> serveLiveCommittedVersion(Reference<MasterData> self) {
 			when(ReportRawCommittedVersionRequest req =
 			         waitNext(self->myInterface.reportLiveCommittedVersion.getFuture())) {
 				self->minKnownCommittedVersion = std::max(self->minKnownCommittedVersion, req.minKnownCommittedVersion);
+				if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
+					for (auto it = req.ssVersionVector.get().versions.begin();
+					     it != req.ssVersionVector.get().versions.end();
+					     it++) {
+						self->ssVersionVector.setVersion(it->first, it->second);
```

Comment:
We need the wait-for-prev logic to be ported to this branch, else the commit version given here could be less than "ssVersionVector->maxVersion", right?

Reply:
yes- I'll open a separate PR for that this week.